www/src/layouts/Single.tsx

```diff
@@ -8,8 +8,8 @@ interface ISingleLayoutProps {
 
 const SingleLayout = ({ children }: ISingleLayoutProps): JSX.Element => (
   <chakra.div id="page">
-    <Header />
-    <Container minH="67.85vh" id="content">
+    <Container minH="74vh" id="content">
```

Comment:
Need to look into how other sites keep their footer down. This is a little hacky.